src/app/api/verifyPin/route.ts

```diff
@@ -0,0 +1,24 @@
+import * as verify from "./verifyPin.api";
+
+export async function POST(request: Request) {
+  try {
+    // Ensure the Content-Type is 'application/json' for correct parsing
+    if (request.headers.get("Content-Type") !== "application/json") {
+      throw new Error("Invalid Content-Type");
+    }
+    const body = await request.json();
+    const captainId = body.captainId;
```

Comment:
The backend doesnt know about captains, this is just itemId

and is not verifyCaptainPin, just verifyValue (for example)
tomorrow can be password 

Just a naming concern